uacp/endpoint_test.go

```diff
@@ -38,11 +38,12 @@ func TestResolveEndpoint(t *testing.T) {
 			"",
 		},
 		{ // Valid, hostname resolved
-			"opc.tcp://localhost:4840/foo/bar",
+			// note: xip.io is hosted by Basecamp (see: https://github.com/basecamp/xip-pdns)
```

Comment:
This is awesome. I never knew.

Reply:
I suppose this should be in an integration test rather than a normal unit test since it requires a mostly working network stack. :shrug:

Reply:
xip is dead. long live https://sslip.io/